scripts/utils/get_coordinates.py

```diff
@@ -0,0 +1,122 @@
+    'fiware-servicepath': '/subservice',
+    'x-auth-token': 'token'
+}
+
+page_size = 500
+
+# END of the configuration part (don't touch below this line ;)
+##############################################################
+
+
+def get_ngsiv1_location(entity):
+    """
+    Search for NGSIv1-style location, i.e. a "location" metadata in some of the attributes of the given entity.
+
+    FIXME: currently metadata location doesn't appear in NGSIv2 results. This method could be pretty useless
+    until https://github.com/telefonicaid/fiware-orion/issues/3122 gets solved, but we leave here for the future
+    in any case.
+
+    :param entity: the entity in which search location
+    :return: the attribute which holds location, 'None' if no one was found
+    """
+    for field in entity:
+        if field == 'id' or field == 'type':
+            continue
+        for m in entity[field]['metadata']:
+            if m == 'location':
+                return field
+    return None
+
+
+def get_ngsiv2_location(entity):
+    """
+    Search for NGSIv2-style location, i.e. a "geo:point" attribute in the given entity.
+
+    :param entity: the entity in which search location
+    :return: the attribute which holds location, 'None' if no one was found
+    """
+    for field in entity:
+        if field == 'id' or field == 'type':
+            continue
+        if entity[field]['type'] == 'geo:point':
+            return field
+    return None
+
+
+def process_page(entities):
+    """
+    Process a page of entities
+
+    :param entities: page of entities (a list) to be processed
+    """
+    for entity in entities:
+        loc_attr = get_ngsiv2_location(entity)
+        if loc_attr is None:
+            loc_attr = get_ngsiv1_location(entity)
+        if loc_attr is not None:
+            print '%s: %s' % (entity['id'], entity[loc_attr]['value'])
+
+
+### Main program starts here ###
+
+# Get first bunch of data
+res = requests.get('%s/v2/entities?offset=0&limit=%s&options=count' % (cb_endpoint, str(page_size)),
+                   headers=headers, verify=False)
+if res.status_code != 200:
+        print 'Error getting entities (%d): %s' % (res.status_code, res.json())
+        sys.exit(1)
+
+# Get count
+count = int(res.headers['fiware-total-count'])
+
+# Process the response
+process_page(res.json())
+
+# Number of batches
+pages = count / page_size
+for i in range(0, pages):
+    offset = (i + 1) * 10
+    res = requests.get('%s/v2/entities?offset=%s&limit=%s' % (cb_endpoint, str(offset), str(page_size)),
+                       headers=headers, verify=False)
+    if res.status_code != 200:
+        print 'Error getting entities (%d): %s' % (res.status_code, res.json())
+        sys.exit(1)
+    process_page(res.json())
+
+print 'Hints:'
+print '* use "awk -F \': \' \'{print $2}\'" filter if you want to grab just the coordinates'
+print '* copy-paste output to https://www.darrinward.com/lat-long so see where you entities are on a map'
```

Comment:
EOL expected

Reply:
Fixed in 8514d10cf4465c727c2b52bd48c8b75f658ce2de